src/FloatConfig.res

```diff
@@ -0,0 +1,73 @@
+type erc20 = {
+  address: string,
+  symbol: string,
+  iconUrl: option<string>,
+}
+
+type contract = {
+  address: string,
+  description: option<string>,
+  codeUrl: option<string>,
+  abiUrl: option<string>,
+}
+
+type contracts = {
+  longShort: contract,
+  staker: contract,
+  stakeBalance: contract,
+  tokenFactory: contract,
+  floatToken: contract,
+  floatCapital_v0: contract,
+  treasury_v0: contract,
+  gems: contract,
+  gemCollectorNft: contract,
+}
+
+type paymentToken = {
+  address: string,
+  isUSDBased: bool,
+  decimals: int,
+  symbol: string,
+  iconUrl: option<string>,
+}
+
+type oracle = {
+  decimals: int,
+  heartbeat: int,
+}
+
+type market = {
+  index: int,
+  name: string,
+  description: string,
+  iconUrl: option<string>,
+  leverage: int,
+  paymentTokenSymbol: string,
+  symbol: string,
```

Comment:
NB: https://github.com/Float-Capital/monorepo/pull/2693/files#diff-7cd2cddf1cc85af0b981ea0ccda872ae31c29b262fe30503ebc2ff7c67247966R88-R90

we also need the market symbol, for examlpe with the flipp3ning it is % (not $).

Reply:
Yea good point. Will add